webservices/legal_docs/advisory_opinions.py

```diff
@@ -83,6 +82,14 @@ def load_advisory_opinions(from_ao_no=None):
         ao_count += 1
     logger.info("%d advisory opinions loaded", ao_count)
 
+def ao_stage_to_pending(stage):
+    if stage == 'Pending':
+        is_pending = True
+    else:
+        is_pending = False
+
+    return is_pending
```

Comment:
In idiomatic `python`, the entire body of this function would be:
`return stage == 'Pending'`